core/arch/arm/plat-sam/conf.mk

```diff
@@ -45,6 +58,14 @@ $(call force,CFG_CORE_DYN_SHM,n)
 
 CFG_MMAP_REGIONS ?= 24
 
+ifeq ($(CFG_SAMA7G5),y)
+CFG_SHMEM_START ?=0x61000000
+CFG_SCMI_SHMEM_START ?= 0x61400000
+CFG_DT_ADDR ?= 0x61500000
+CFG_ATMEL_SFC ?= n
+CFG_NVMEM_DIE_ID ?= n
+CFG_NVMEM_HUK ?= n
+endif
```

Comment:
I was expecting `else` here to cover all the variables above for the other case, something like:
```
else
CFG_SHMEM_START = 0x21000000
CFG_SCMI_SHMEM_START  ?= 0x21400000
...
CFG_NVMEM_HUK ?= y
endif
```
to make it easier to see the difference.

Reply:
Calculated the addresses in Makefile and updated for the other case, thanks.